net/interface.go

```diff
@@ -1,28 +1,20 @@
-package vnet
+package net
```

Comment:
It might be better to avoid using package name of stdlib.

Reply:
I think @at-wat has a good point.
I have thought about this for a while... I'd like to avoid `vnet` for the interface package. How about `transport`? It's "mostly" about transport layer, and sort of promoting @stv0g 's "Net" interface to be the part of the primary package of this repo.

Reply:
@enobufs I agree :) I will implement this change